include/swift/Option/Options.td

```diff
@@ -890,7 +890,11 @@ def no_static_executable : Flag<["-"], "no-static-executable">,
 
 def use_ld : Joined<["-"], "use-ld=">,
   Flags<[DoesNotAffectIncrementalBuild]>,
-  HelpText<"Specifies the linker to be used">;
+  HelpText<"Specifies the flavor of the linker to be used">;
+
+def ld_path : Joined<["--"], "ld-path=">,
+  Flags<[DoesNotAffectIncrementalBuild, ArgumentIsPath]>,
```

Comment:
I think it may be a good idea to have this also be `HelpHidden`. Since I'm not sure there's much value to just `swiftc` users here, but I'll defer this to you.

Reply:
will do — @compnerd requested the same thing